Libplanet/Blockchain/BlockChain.cs

```diff
@@ -111,7 +118,7 @@ public Block<T> Tip
         }
 
         public void Validate(
-            IEnumerable<Block<T>> blocks,
+            IReadOnlyList<Block<T>> blocks,
```

Comment:
As this is a `public` method, this change also should be written in the changelog.